src/registrar/config/urls.py

```diff
@@ -114,18 +118,15 @@
     ),
 ]
 
-
+# What is the purpose of this?
+# This behaviour gets overwritten, so this doesn't do anything...
+# Login in particular
```

Comment:
**comment (non-blocking)**: I think you're right that we could remove the conditional and make these redirects apply in all situations.